projects/components/src/file-upload/file-display/file-display.component.ts

```diff
@@ -50,23 +59,16 @@ export class FileDisplayComponent {
   public file?: File;
 
   @Input()
-  public state: FileUploadState = FileUploadState.NotStarted;
-
-  @Input()
-  public showState: boolean = false;
-
-  @Input()
-  public errorStateTooltipText: string = '';
+  public latestUploadEvent?: FileUploadEvent<unknown>;
 
   @Output()
   public readonly deleteClick: EventEmitter<void> = new EventEmitter();
 
-  // Only show file delete option if file upload has not started or it has resulted in failure
-  public get showDelete(): boolean {
-    return this.state === FileUploadState.NotStarted || this.state === FileUploadState.Failure;
```

Comment:
Now we're also showing the delete when the upload is in progress? I don't think we should do that.
Because if a big file upload is in progress and then what happens if a user removes it in between. After the success(Only if we have a way to delete it later) and before the start is fine showing that.

Reply:
added back